dipy/tracking/streamline.py

```diff
@@ -2,164 +2,18 @@
 from warnings import warn
 import types
 
-from distutils.version import LooseVersion
 from scipy.spatial.distance import cdist
 
 import numpy as np
-import nibabel as nib
 from nibabel.affines import apply_affine
-from dipy.tracking.streamlinespeed import set_number_of_points
-from dipy.tracking.streamlinespeed import length
+from nibabel.streamlines import ArraySequence as Streamlines
+from dipy.tracking.streamlinespeed import (set_number_of_points, length,
+                                           compress_streamlines)
 from dipy.tracking.distances import bundles_distances_mdf
-from dipy.tracking.streamlinespeed import compress_streamlines
 import dipy.tracking.utils as ut
-from dipy.tracking.utils import streamline_near_roi
```

Comment:
What is the preferred convention here?

-  import as ut,  or 
- import : move_streamlines, streamline_near_roi, reduce_rois

Reply:
No strong opinion here, but I will change it to `import : move_streamlines, streamline_near_roi, reduce_rois`

Reply:
I think that either is fine. Some (me included) prefer the `import utils as ut` and then calling everything with a `ut.this` and `ut.that`, because it's more explicit. It tells you each time exactly where that function comes from.